src/main/resources/application-folio.yml

```diff
@@ -10,6 +10,8 @@ folio:
     feign:
       enabled: true
   kafka:
+    retry-interval-ms: 1000
```

Comment:
If we activate folio.retry.enabled property to use provided by folio RetryTemplate, we could omit this setting, because it's the same as default - org.folio.spring.tools.kafka.FolioKafkaProperties#retryIntervalMs

Reply:
Done. removed